packages/eslint-plugin-gestalt/src/eslintASTHelpers.js

```diff
@@ -0,0 +1,313 @@
+  // Look in local scope for variable reference
+  const variableNode = scope.references.find((reference) => reference.identifier.name === name);
+  return variableNode;
+};
+
+/* ↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓  getComponentNameFromAttribute ↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
+This function returns the component's name containing the attribute's node (nodeAttribute).
+Examples:
+<div {...props} /> >> returns div for the spread props attribute
+*/
+type GetComponentNameFromAttributeType = ({| nodeAttribute: GenericType |}) => string;
+
+export const getComponentNameFromAttribute: GetComponentNameFromAttributeType = ({
+  nodeAttribute,
+}) => nodeAttribute.parent.name.name;
+
+/* ↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓  hasImport ↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
+This function checks is a given node (importNode) contains a given import path (path), and returns true if so.
+Examples:
+import { Box } from 'gestalt' >> path="gestalt"
+import { Box } from 'app/box' >> path="app/box"
+*/
+type HasImportType = ({| importNode: GenericType, path: string |}) => boolean;
+
+export const hasImport: HasImportType = ({ importNode, path }) => {
+  const importName = importNode.source ? importNode.source.value : null;
+  return importName === path;
+};
+
+/* ↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓  updateGestaltImportFixer ↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
+This function returns an array of arrays containing the named imports ([imported name, local or aliased name]) from a node (importNode).
+*/
+type GetNamedImportsComponentsType = ({| importNode: GenericType |}) => $ReadOnlyArray<
+  $ReadOnlyArray<string>,
+>;
+
+export const getNamedImportsComponents: GetNamedImportsComponentsType = ({ importNode }) => {
+  const namedImports = importNode.specifiers.map((node) => [node.imported.name, node?.local?.name]);
+  return namedImports;
+};
+
+/* ↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓  isTag ↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
+This function checks is a given node (elementNode) contains a given tag (tagName), and returns true if so.
+Examples:
+<div /> >> if tagName="div" returns true
+<div /> >> if tagName="button" returns false
+*/
+type IsTagType = ({| elementNode: GenericType, tagName: string |}) => boolean;
+
+export const isTag: IsTagType = ({ elementNode, tagName }) => elementNode?.name?.name === tagName;
+
+/* ↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓  hasLonelyAttribute ↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
+This function checks is a given tag (tagName) in a node (elementNode) contains only a single attribute (attribute), and returns true if so.
+Examples:
+<div ref={} /> >> if attribute="ref" returns true
+<div ref={} style={} /> >> if attribute="ref" returns false
+*/
+type HasLonelyAttributeType = ({|
+  elementNode: GenericType,
+  tagName: string,
+  attribute: string,
+|}) => boolean;
+
+export const hasLonelyAttribute: HasLonelyAttributeType = ({ elementNode, tagName, attribute }) => {
+  return (
+    isTag({ elementNode, tagName }) &&
+    elementNode.attributes.length === 1 &&
+    elementNode.attributes[0].name &&
+    elementNode.attributes[0].name.name === attribute
+  );
+};
+
+/* =================  FIXERS =================
```

Comment:
Even if we don't split this out to one function : one file, we should at least separate the helpers from the fixers

Reply:
DONE